### PR TITLE
Datadog.Agent version 7.31.0

### PR DESCRIPTION
**Pull request by julien-lebot**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/27714)

### CODE REVIEW

File: manifests/d/Datadog/Agent/7.31.0/Datadog.Agent.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Datadog.Agent
+PackageVersion: 7.31.0
```
Comment (OfficialEsco) [inline manifests/d/Datadog/Agent/7.31.0/Datadog.Agent.installer.yaml:5]:
```suggestion
PackageVersion: 7.31.0.1
```

File: manifests/d/Datadog/Agent/7.31.0/Datadog.Agent.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Datadog.Agent
+PackageVersion: 7.31.0
```
Comment (OfficialEsco) [inline manifests/d/Datadog/Agent/7.31.0/Datadog.Agent.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 7.31.0.1
```

File: manifests/d/Datadog/Agent/7.31.0/Datadog.Agent.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Datadog.Agent
+PackageVersion: 7.31.0
```
Comment (OfficialEsco) [inline manifests/d/Datadog/Agent/7.31.0/Datadog.Agent.yaml:5]:
```suggestion
PackageVersion: 7.31.0.1
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by OfficialEsco** [comment]
PS: you need to change the version folder name

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**